internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -89,7 +89,7 @@ var (
 				ToolingVersion:          "0.1.0",
 			},
 			AddonsVersion: AddonsVersion{
-				Cilium:  &AddonVersion{"1.5.3", 1},
+				Cilium:  &AddonVersion{"1.5.7", 0},
```

Comment:
last update we did bump all the kubernetes versions below so they all had the same cilium version. Should we do the same here?